client/src/Components/ReviewsContainer/List/ReviewList.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from "react"
+import Container from "react-bootstrap/Container"
+import Row from "react-bootstrap/Row"
+import Col from "react-bootstrap/Col"
+import Button from "react-bootstrap/Button"
+import ReviewTile from "./ReviewTile.jsx"
+import useReviews from "../../../contexts/ReviewsContext.jsx"
+
+const ReviewList = (props) => {
+  const { reviews, fetchReviews } = useReviews()
+  const [page, setPage] = useState(1)
+  const [count, setCount] = useState(2)
+  const [sort, setSort] = useState("relevant") // based on sort
+  const [product_id, setProduct_id] = useState(42367)
```

Comment:
this would later be the ID from ProductsContext right?

Reply:
yep!